File: 0x01-python-if_else_loops_functions/1-last_digit.py
#!/usr/bin/python3
import random
number = random.randint(-10000, 10000)
x = number % 10
if x  > 5:
    print("Last digit of {0} is {1} and is greater than 5".format(number, x))
elif x == 0:
    print("Last digit of {0} is {1} and is 0".format(number, x))
elif 0 != x < 6:
    print("Last digit of {0} is {1} and is less than 6 and not 0".format(number, x))

File: 0x01-variables_if_else_while/1-last_digit.c
#include <stdlib.h>
#include <time.h>
#include <stdio.h>

/**
 * main - entry point
 * description: last digit
 * Return: 0
 */

int main(void)
{
	int n;
	int m
	m = n % 10;

	srand(time(0));
	n = rand() - RAND_MAX / 2;
	if (n > 5)
	{
		printf("Last digit of %d is %d and is greater than 5\n", n , m);
	}
	else if (n == 0)
	{
		printf("Last digit of %d is %d and is 0\n", n , m);
	}
	else if (n < 6 && n != 0)
	{
		printf("Last digit of %d is %d and is less than 6 and not 0\n", n , m); 
	}
	return (0);
}